eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Snapshots.hs

```diff
@@ -1,7 +1,12 @@
 {-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE RankNTypes #-}
-{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
Why is orphans warning being turned off? There is no reason why `HasOKey` instance should not live right next to the type definition
```suggestion
```

Reply:
Got it!